corehq/apps/app_manager/static/app_manager/js/case-config-ui.js

```diff
@@ -650,11 +649,24 @@ hqDefine('app_manager/js/case-config-ui.js', function () {
             self.isBlank = ko.computed(function () {
                 return !self.key() && !self.path();
             });
-            self.description = ko.computed(function () {
-                var config = self.case_transaction.caseConfig;
-                var type = config.descriptionDict[self.case_transaction.case_type()];
-                if (type) {
-                    return type[self.key()] || '';
+            self.caseType = ko.computed(function () {
+                return self.case_transaction.case_type();
+            });
+            self.updatedDescription = ko.observable('');
+            self.description = ko.computed({
+                read: function () {
+                    if (self.updatedDescription()) {
+                        return self.updatedDescription();
+                    }
+                    var config = self.case_transaction.caseConfig;
+                    var type = config.descriptionDict[self.caseType()];
+                    if (type) {
+                        return type[self.key()] || '';
+                    }
+                },
+                write: function (value) {
+                    self.updatedDescription(value);
+                    $('.read-only').data('bs.popover').options.content = value;
```

Comment:
I did some further testing and based on another change i made, this seems to be unnecessary now but for reasons I cant totally explain it was not updating all of the popovers. Unmodified popovers were keeping the correct value